src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
+
+    line-height: 15px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    line-height: 13px;
+    font-size: 11px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 15px;
+    margin-bottom: 20px;
+  }
+
+  &__stars--4 {
+    display: flex;
+  }
+
+  .stars__star {
```

After changing classes on <div class="Card__star"> you can use here and another places new class name
```suggestion
  .Card__star {
```